README.md

```diff
@@ -130,7 +130,7 @@ As can be seen in the sample app, there are just few step required to integrate
            try {
                Kin.launchMarketplace(MainActivity.this);
                 System.out.println("Public address : " + Kin.getPublicAddress());
-                } catch (TaskFailedException e) {
+                } catch (ClientException e) {
```

Comment:
do we need to document what kin of error can be returned and what are this meaning?

Reply:
it's documented on `ClientException`